lib/backend/syncersv1/updateprocessors/networkpolicyprocessor.go

```diff
@@ -67,7 +67,24 @@ func convertPolicyV2ToV1Spec(spec apiv2.PolicySpec) (interface{}, error) {
 		PreDNAT:       spec.PreDNAT,
 		Types:         policyTypesAPIV2ToBackend(spec.Types),
 	}
-
+	if len(spec.Types) == 0 {
```

Comment:
I'm not 100% sure of this.  Previously, we did this defaulting on write but this is on the syncer's read path.  Previously, on the read path, we defaulted to `[ingress, egress]`, I think.

Reply:
Do you know where the write path for v2 is?